pom.xml

```diff
@@ -19,11 +19,12 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
         <build.counter>1</build.counter>
-        <oap.deps.maven.surefire.version>2.22.2</oap.deps.maven.surefire.version>
-        <oap.deps.maven.checkstyle.version>3.1.1</oap.deps.maven.checkstyle.version>
-        <oap.deps.checkstyle.version>8.39</oap.deps.checkstyle.version>
-        <oap.deps.maven.source.version>3.0.1</oap.deps.maven.source.version>
-        <oap.deps.maven.flatten.version>1.1.0</oap.deps.maven.flatten.version>
+        <oap.deps.maven.surefire.version>3.0.0-M5</oap.deps.maven.surefire.version>
+        <forkCount>2</forkCount>
```

Comment:
Or consider using `1C` value (the same number of maven builds as CPU on the machine).